rabbitmq/docker-command-smile-traefik.sh

```diff
@@ -18,9 +18,14 @@ export CILOGON_CLIENT_ID=<<cilogon id>>
 export CILOGON_CLIENT_SECRET=<<cilogon client secret>>
 export CILOGON_CALLBACK_URL=<<ci logon callback url>>
 
+# if the minio-api.server doesn't work, it should ip address and port
```

Comment:
typo. It should "be" ip address and port :D

Reply:
updated